integration_tests/suite/base/test_meeting.py

```diff
@@ -445,7 +466,8 @@ def test_bus_events_progress(_, me, meeting):
 @fixtures.ingress_http()
 @fixtures.meeting()
 @fixtures.meeting()
-def test_purge_old_meetings(_, meeting_too_old, meeting_too_young):
+@fixtures.meeting(persistent=True)
+def test_purge_old_meetings(_, meeting_too_old, meeting_too_young, meeting_persistent):
     too_old = datetime.now() - timedelta(hours=72)
     with db.queries() as queries:
         queries.set_meeting_creation_date(meeting_too_old['uuid'], too_old)
```

Comment:
The test is invalid if creation date is not changed for persistent meeting. Otherwise young and persistent meetings will be ignored by the creation date condition and not persistent
```suggestion
        queries.set_meeting_creation_date(meeting_too_old['uuid'], too_old)
        queries.set_meeting_creation_date(meeting_persistent['uuid'], too_old)
```